viewing_party/party.py

```diff
@@ -1,23 +1,179 @@
+from collections import Counter
+from shutil import move
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {
+        "title":title,
+        "genre":genre,
+        "rating":rating
+    }
+
+    for val in movie_dict.values():
+        if not val:
+            return None
+
+    return movie_dict
+
+def add_to_watched(user, movie):
+
+    for key in user:
+        user[key].append(movie)
+    
+    return user
+
+def add_to_watchlist(user, movie):
+
+    for key in user:
+        user[key].append(movie)

```

Comment:
Similar to my comment above, instead of iterating through all the keys and appending movie, we can directly append the movie the watchlist with `user["watchlist"].append(movie)` and remove the for loop.